src/nicolai_munsterhjelm_ex/ex05/myrand.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Nicolai Munsterhjelm'
+__email__ = 'nicolai.munsterhjelm@nmbu.no'
+
+
+class LCGRand:
+    def __init__(self, seed):
+        """
+        Initialise a linear congruence random number generator.
+
+        Arguments
+        ---------
+        seed : int
+            The initial seed for the generator
+        """
+        self.hidden_state = seed
+        self.a = 7**5
+        self.m = 2**31 - 1
+
+    def rand(self):
+        """Generates a random number using the LCG method,
+         based on the seed."""
+        self.hidden_state = (self.a * self.hidden_state) % self.m
+        return self.hidden_state
+
+    def random_sequence(self, length):
+        """Generates an iterable sequence of random numbers
+        with the specified length."""
```

Comment:
correct wording is "an iterator" An iterable sequence would be something like a list or a tuple.